ipaclient/plugins/server.py

```diff
@@ -12,6 +12,12 @@
 @register(override=True, no_fail=True)
 class server_del(MethodOverride):
     def interactive_prompt_callback(self, kw):
+        server_list = []
+        try:
+            server_list = kw['cn']
+        except Exception as e:
```

Comment:
1. Can you please use KeyError?
2. Please remove 'as e'. e is unused variable
3 you can use
```
server_list = kw.get('cn')
if not server_list:
    raise err......
```